### PR TITLE
firefox fails to compute style for element without position

### PR DESCRIPTION
**Pull request by schhunchha**

### CODE REVIEW

File: slick.grid.js [hunk 1]
```diff
@@ -2033,8 +2033,9 @@ if (typeof Slick === "undefined") {
                 return;
             }
 
-            cacheEntry.rowNode[0].parentElement.removeChild(cacheEntry.rowNode[0]);
-
+            if(cacheEntry.rowNode) {
+                cacheEntry.rowNode[0].parentElement.removeChild(cacheEntry.rowNode[0]);
+            }
```
Comment (wesamhaboush) [inline slick.grid.js]: what was this necessary?

File: slick.grid.js [hunk 2]
```diff
@@ -2195,8 +2196,7 @@ if (typeof Slick === "undefined") {
                 'top': $paneHeaderL.height(), 'height': paneTopH
             });
 
-            var paneBottomTop = $paneTopL.position().top
-                + paneTopH;
+            var paneBottomTop = $paneTopL.css('position') ? $paneTopL.position().top : paneTopH + paneTopH;
```
Comment (wesamhaboush) [inline slick.grid.js]: can we make the precedence clearer here? i do not know actually how this calculated:
```
 var paneBottomTop = ($paneTopL.css('position') ? $paneTopL.position().top : paneTopH) + paneTopH;
```

Comment (wesamhaboush) [reply slick.grid.js]: or even better:
```
 var paneBottomTop = getPositionTop() + paneTopH;
```

File: slick.grid.js (hunk 1 above)
Comment (schhunchha) [reply slick.grid.js]: not for this one. something I had found to be done in past. reduce noise on console.

File: slick.grid.js (hunk 2 above)
Comment (schhunchha) [reply slick.grid.js]: updated!

File: slick.grid.js (hunk 1 above)
Comment (wesamhaboush) [reply slick.grid.js]: cool. keep it. let's go :) .. well done on finding the issue :)

**Pull request merged**